### PR TITLE
fix: (strf-5280) Multiple themes - Changes to config.json not reflected

### PR DESCRIPTION
**Pull request by jairo-bc**
#### What?
On config.json current variant is being reset.

#### Tickets / Documentation

https://jira.bigcommerce.com/browse/STRF-5280

### CODE REVIEW

File: bin/stencil-start.js [hunk 1]
```diff
@@ -160,6 +160,18 @@ async function startServer() {
         }
     });
 
+    Bs.watch('config.json', (event) => {
+        if (event === 'change') {
+            try {
+                themeConfig.resetVariationSettings();
```
Comment (MaxGenash) [inline bin/stencil-start.js:166]: Just curious when `themeConfig.resetVariationSettings()` can throw errors?

Comment (jairo-bc) [reply bin/stencil-start.js]: haha, never :) That was left from previous trials using `updateConfig()`

**Review by MaxGenash — APPROVED** [review]

File: lib/theme-config.js [hunk 2]
```diff
@@ -121,6 +121,15 @@ ThemeConfig.prototype.setThemePath = function (themePath) {
...
 
+/**
+ * Reset Current Variation Settings
+ *
+ * @returns {ThemeConfig}
+ */
+ThemeConfig.prototype.resetVariationSettings = function () {
+    this.currentVariationSettings = null;
+};
+
```
Comment (junedkazi) [inline lib/theme-config.js:132]: do we need to return something here ?

Comment (jairo-bc) [reply lib/theme-config.js:132]: Nope. It's just a setter. Some methods here return `this` for chaining, but I haven't found any occurrences of this.

**Review by junedkazi — APPROVED** [review]

**Pull request merged**